nx.json

```diff
@@ -10,6 +10,16 @@
   "targetDependencies": {
     "release:phase2": [{"target": "release:phase2", "projects": "dependencies"}]
   },
+  "targetDefaults": {
+    "build": {
+      "dependsOn": ["^build"],
+      "outputs": [
+        "{projectRoot}/lib",
+        "{projectRoot}/dist",
+        "{projectRoot}/tsconfig.build.tsbuildinfo"
+      ]
+    }
+  },
```

Comment:
https://nx.dev/reference/project-configuration for global references, line by line comment inc